cmd/eibc/init/init.go

```diff
@@ -41,6 +41,8 @@ func Cmd() *cobra.Command {
 		Use:   "init",
 		Short: "Initialize eibc client",
 		Run: func(cmd *cobra.Command, args []string) {
+			pterm.Warning.Println("warning! by default roller is using a public endpoint which is not reliable. for production usage it's highly recommended to use a private endpoint. A freemium private endpoint can be obtained in the following link https://blastapi.io/chains/dymension")
+			pterm.Warning.Println("warning! Hub private endpoints can be updated anytime after initial setup by running roller config set hub-rpc-endpoint <private-endpoint>")
```

Comment:
when suggesting a command, please highlight it in the output, like in:

https://github.com/dymensionxyz/roller/blob/main/cmd/eibc/init/init.go#L268-L272